nix/test-environments/example-setup-03.nix

```diff
@@ -72,6 +70,22 @@ in
               peg-tolerance-percentage = 0.000001; # 0.000001% tolerance assures that the price will not be pegged
             }
           ];
+          contracts = [
+            {
+              name = "AggregatedDataFeedStore";
+              address = upgradeableProxyADFSContractAddressInk;
+              creation-byte-code = null;
+              deployed-byte-code = null;
+              contract-version = 2;
+            }
+            {
+              name = "multicall";
+              address = "0xcA11bde05977b3631167028862bE2a173976CA11";
+              creation-byte-code = null;
+              deployed-byte-code = null;
+              contract-version = 3;
+            }
+          ];
```

Comment:
When I remove the `multicall` contract from the config, the sequencer still manages to parse the rounds. How is that possible?

Reply:
Ahaa.. `load_round_counters_from_chain` does not use multicall

Reply:
So, what do we use `multicall` for?

Reply:
Initial implementation both round counters and history values used multicall. Then I switched the reading of round counters with faster ( in theory ) call `get_storage_at`. Maybe the same is possible for reading the historical values in the ADFS contract. In the legacy contract, we still use mullticall for reading historical values.